packages/generator-one-app-module/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@americanexpress/generator-one-app-module",
+  "version": "6.0.0",
+  "description": "generator for One App Modules",
+  "license": "Apache-2.0",
+  "contributors": [
+    "Andres Escobar <Andres.Escobar@aexp.com> (https://github.com/anescobar1991)",
+    "James Singleton <James.Singleton1@aexp.com> (https://github.com/JamesSingleton)",
+    "Jimmy King  <Jimmy.King@aexp.com> (https://github.com/10xLaCroixDrinker)",
+    "Jonathan Adshead <Jonathan.Adshead@aexp.com> (https://github.com/JAdshead)",
+    "Michael Tobia <Michael.M.Tobia@aexp.com> (https://github.com/Francois-Esquire)",
+    "Michael Tomcal <Michael.A.Tomcal@aexp.com> (https://github.com/mtomcal)",
+    "Stephanie Coates  <Stephanie.Coates1@aexp.com> (https://github.com/stephaniecoates)",
+    "Nelly Kiboi <Nelly.J.Kiboi@aexp.com> (https://github.com/nellyk)",
+    "Nickolas Oliver <nickolas.oliver@aexp.com> (https://github.com/PixnBits)"
+  ],
+  "files": [
+    "generators"
+  ],
+  "main": "generators/index.js",
+  "repository": {
```

Comment:
Was there a reason to make the `repository` section different here compared to https://github.com/americanexpress/one-app-cli/blob/master/packages/one-app-bundler/package.json#L12-L15?

Reply:
updated in 1e2900f